CHANGES.md

```diff
@@ -111,6 +92,24 @@ tasks could sometimes get orphaned when an auto restart occurred.
 
 Cylc 8 production-ready release.
 
+### Major Changes
+
+* Python 2 -> 3.
+* Internal communications converted from HTTPS to ZMQ (TCP).
+* PyGTK GUIs replaced by:
+  * Terminal user interface (TUI) included in cylc-flow.
+  * Web user interface provided by the cylc-uiserver package.
+* A new scheduling algorithm with support for branched workflows.
+* Command line changes:
+  * `cylc run` -> `cylc play`
+  * `cylc restart` -> `cylc play`
+  * `rose suite-run` -> `cylc install; cylc play <id>`
+* The core package containing Cylc scheduler program has been renamed cylc-flow.
+* Cylc review has been removed, the Cylc 7 version remains Cylc 8 compatible.
+* [New documentation](https://cylc.github.io/cylc-doc/stable).
+
+See the [migration guide](https://cylc.github.io/cylc-doc/stable/html/7-to-8/index.html) for a full list of changes.
+
 ### Enhancements
 
 [#4964](https://github.com/cylc/cylc-flow/pull/4964) -
```

Comment:
Is it worth keeping the enhancements and fixes sections of 8.0.0, considering that they only really make sense in the context of 8.0rc3?

Reply:
I think it makes sense to leave in the first 8.0.0 release for context, it's an arbitrary breakpoint anyhow.